src/backend/mailbox_manager/utils/dimail.py

```diff
@@ -71,6 +71,48 @@ def get_headers(self, user_sub=None):
 
         return self.pass_dimail_unexpected_response(response)
 
+    def send_domain_creation_request(self, domain_name, request_user):
+        """Send a domain creation request to dimail API."""
+
+        payload = {
+            "domain": domain_name,
+            "context": domain_name,  # for now, we put each domain on its own context
+            "features": ["webmail", "mailboxes"],
+        }
+        try:
+            response = session.post(
+                f"{self.API_URL}/domains/",
+                json=payload,
+                headers={"Authorization": f"Basic {self.API_CREDENTIALS}"},
+                verify=True,
+                timeout=10,
+            )
+        except requests.exceptions.ConnectionError as error:
+            logger.error(
```

Comment:
As `error` is an exception, why don't you use logger.exception?

Reply:
Absolutely, I just didn't know there was a dedicated function but that's even better!

About this very line; I might change it for the error to be logged but the response to be passed as is (without exception).